chatkit/src/main/java/com/stfalcon/chatkit/messages/MessageTextUtils.kt

```diff
@@ -22,7 +22,7 @@ import java.util.regex.Pattern
 class MessageTextUtils {
 
 
-    companion object {
+        companion object {
```

Comment:
why?

Reply:
idk, some random tab press, didn't see that